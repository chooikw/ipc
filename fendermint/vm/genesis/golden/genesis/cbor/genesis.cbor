a96a636861696e5f6e616d656e040bc2ae2334e2808fc2932dc2ad6974696d657374616d701ba0b9a3d965ef92ed6f6e6574776f726b5f76657273696f6e1568626173655f666565510013aa3eef6d427cf40466e76af9c788786b706f7765725f7363616c65006a76616c696461746f727389a26a7075626c69635f6b657958410496f092fcdb22919aab509aa4baf051bfdb1f7c0616f038edd1407c884661b7de16f08e3e82bd30d9b881a95177b7005d64343d281acbc5de95701dd3140359a565706f7765725100ece6223bf395cef80edf1cc633a159d3a26a7075626c69635f6b65795841045222ee4e3c45179b991311f5d74de756f610ce4ddb0f7b7ab9fe445291033bb318b1680535983a225cbdcd0daa057c7bd487639af90b0b8dcd94003ee45ebeeb65706f77657251002365102f6bc73b742a19f16d1a81f47fa26a7075626c69635f6b6579584104a9a0def5cca3b8f3ebc78c92b5ca3c0f0f221baedf9a4071cf6ff4670bf720e9e3e8f78d52c94fb03d3bc99fc76c37513fa240a45e5d8d0d2e9a01ea0df7337965706f7765725100f885e58da3c2f8775d89bdcfd877463ca26a7075626c69635f6b6579584104663fbfc331cca92afad975e26d52e0bf0eee136b11ad2765bc1c09ef8e1814fa7617c9d755c052f8d93064d852a1054c0843547118ee7429d4ba704d152d2f4465706f7765725100a0b08d1e0d2076222d9356aad6f574e6a26a7075626c69635f6b65795841047969f4f568404af7898d4485da4787b7a966df6e0398af1d6d06dc5742f0b16684e99732f7bd4d56be46bd70e94d64138d7b0971d6f7ef2a5a1114f71666da4c65706f7765725100d8f7cab4b4753ccb1f0fa30a7bb2ba8ea26a7075626c69635f6b65795841040c1b0e58d1b457729ef3ac90c5ba5d6432d0d08551c6a3af3a573bf7d15e347f515cfebdd4bb0cc7e2881a09f0374a637867b68e7da63698156bd6643a3029a465706f7765724900e394eac3bec07f55a26a7075626c69635f6b6579584104272ea3a4a7694c1ee8c7bae9456891668da95bf1666a0525db04194d072e140037560f684420a44830b6c56c29e5dbaa495fd25b92c0aa74ca28aa040f66dfd065706f7765725100caf94566da4b90e869814030da523ee8a26a7075626c69635f6b6579584104824015417b4f865c2ada4316959d8966340420c1f51083eee955c6e1152fdb3b7bf9ef83dc6fe531bd655483be72fb30555e614096ed923987453ace5287752d65706f77657240a26a7075626c69635f6b6579584104852e2fd862d0d4d605f96d54d08431dba9c280b08712e31128e8d918b398d730f4d039835070b8e439c6a78f36f32a8e8aa943ed4fe6b46a491bfc117d60601165706f7765725100f39f8ecc0feef6f91ef0e6e85dd9cf65686163636f756e747387a2646d657461a1684d756c7469736967a4677369676e65727385550190d12fdeb755b31e147d271456db2007485b164c55018f90a17f6cbc6a8da40a8ab431765af63c0ad0ab5501f802e9f33e63c3ae61395cbff809022fb1cb3fc35501859dab6bb3f0a778239f55ea4b21b01f789d4f9e5501b292ef2bf74ba235d87d672492dfcd527898c810697468726573686f6c64017076657374696e675f6475726174696f6e1bffffffffffffffff6d76657374696e675f73746172741bf715a10352f3aebd6762616c616e63655100b61433b86e731bbc3f4f2b5b7dfa59e1a2646d657461a1684d756c7469736967a4677369676e65727384550179974e6e30282d906eb766e4f5c95bd05d8c6ebf5501fe2d06f993d907706dbbad23bb4b30614400f6e45501d1f28fd7fb06ca4f35ba6fef51f7672118b2b3df55014291a91aa6c07f7c0302043911c307166ce7db3f697468726573686f6c64017076657374696e675f6475726174696f6e1b49b54e6ba55374c56d76657374696e675f73746172741b585ed2687281ba516762616c616e636540a2646d657461a1684d756c7469736967a4677369676e657273855501f745e780de71864a70f313626851f3f2c18de92655019f42e37691328884e063690f7546962928b57fde550162d8eeb325f3b598726505361bd80195b3eb902a5501517ea79cc82f78086ac3008e617bd6421aa60f615501f901539acf51df3e3f60b9c1682a48c98213021d697468726573686f6c64047076657374696e675f6475726174696f6e1ba945eca5c3fef4e06d76657374696e675f73746172741bdc1b43570202a59f6762616c616e63655100d607f32a9c8699f91ccd156ad072d701a2646d657461a1684d756c7469736967a4677369676e657273855501dff4ba9af5e5cd77387006e5bd28d7132920e02c55012c0dbe168c904e973bc31b1bfde509760f8773d0550144e12bde00b14049f8dc8be8465b27886374b7b75501978fa79ed649726b6479573f6ef0222dfc18d45b5501f809d234f6076c23d9298f37c35a60ba09beb7d8697468726573686f6c64017076657374696e675f6475726174696f6e1b9f72548148dc17126d76657374696e675f73746172741b7eff08e728dca4266762616c616e636540a2646d657461a1684d756c7469736967a4677369676e6572738355018c6aad2f5e69bb6245fbcd8480b94ba31bcb866b55013a006db6b71efdc6d6705d375821032374146fdd550199e69d310d73e53be8e518de2ce11e120d1accd4697468726573686f6c64027076657374696e675f6475726174696f6e1b5bb35fe0a64a88d36d76657374696e675f73746172741b416e57f99cf8e4696762616c616e636551003a6a5b35b8055bbc139e57928600bf0fa2646d657461a1684d756c7469736967a4677369676e657273825501187e9dd34b1c6743ca267c1b018b383d1b17c2cc5501d8c0d2b7de93dc6335ecb5f32a407475036f0d26697468726573686f6c64027076657374696e675f6475726174696f6e1bffffffffffffffff6d76657374696e675f73746172741b5917fb94a1c495656762616c616e63655100f4598ba3ab456f9fbe93b437c6f60de1a2646d657461a1674163636f756e74a1656f776e657256040ac57fe9b4f8837b895b2079713f37a7c524eb20b66762616c616e63655100769f08cad43b9f9754ed66a8a07d1ce17365616d5f7065726d697373696f6e5f6d6f6465a1646d6f64656c756e7265737472696374656463697063a16767617465776179a4697375626e65745f6964a264726f6f741be6d884c2929ab32d686368696c6472656e8076626f74746f6d5f75705f636865636b5f706572696f641b0688b9ed0b34e72f736d616a6f726974795f70657263656e746167651843776163746976655f76616c696461746f72735f6c696d69741849