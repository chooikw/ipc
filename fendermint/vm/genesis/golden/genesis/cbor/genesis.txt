Genesis { chain_name: "\u{4}\u{b}Â®#4\u{200f}\u{93}-\u{ad}", timestamp: Timestamp(11581468070873371373), network_version: NetworkVersion(21), base_fee: TokenAmount(26139298864604834844.15329121596601356), power_scale: 0, validators: [Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [39958494, 2040337, 59694356, 1596352, 64692092, 45880431, 17410635, 38169261, 50125602, 2472996], magnitude: 1, normalized: true }, y: Field { n: [219557, 488645, 6154583, 10513199, 23344189, 32358423, 1742103, 12805858, 37651133, 375843], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(314892729634477233575.319852355487357395)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [16989107, 59839652, 62368671, 20409405, 49680590, 30636501, 19996509, 6188644, 38681669, 1345723], magnitude: 1, normalized: true }, y: Field { n: [6209259, 16781241, 12115161, 40625196, 64259939, 42032926, 64803034, 15239538, 341400, 404570], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(47047730110817122790.731233985252422783)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [66527465, 66918850, 466166, 45842025, 51323419, 24284931, 7915819, 48484271, 49663139, 2779191], magnitude: 1, normalized: true }, y: Field { n: [32977785, 41974403, 13685481, 43088246, 20947520, 31133140, 62691836, 20889844, 59593417, 3734077], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(330343774330496180857.212299294759405116)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [35132666, 50494435, 41311169, 28067508, 51310099, 55883823, 26697254, 44346347, 63123916, 1675247], magnitude: 1, normalized: true }, y: Field { n: [19738436, 43782981, 54697291, 29647801, 541524, 11026771, 50744709, 21750628, 30889408, 1934834], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(213593185773350208264.313896363854492902)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [49328486, 28775888, 49403600, 28839522, 61433567, 43114989, 13912157, 19652134, 16083008, 1989245], magnitude: 1, normalized: true }, y: Field { n: [40294988, 4537797, 49456545, 29842399, 59603721, 39016708, 7067406, 20273913, 53671869, 2177637], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(288399855777258154354.128991727442377358)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [22951039, 30342644, 37417893, 34948890, 3330256, 24024921, 54184204, 22923899, 39375284, 198339], magnitude: 1, normalized: true }, y: Field { n: [36710820, 49649934, 57246038, 37353112, 58222518, 905880, 8495263, 3350410, 45995195, 1333055], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(16.398990269602627413)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [53351424, 17191745, 5397936, 63281576, 42838363, 22684761, 8105620, 19954595, 61122409, 641960], magnitude: 1, normalized: true }, y: Field { n: [57073616, 36339971, 44518562, 24005378, 38363090, 41514730, 57431746, 43065538, 57164832, 906627], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(269798344701422492711.839715388602597096)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [19913531, 24229957, 4124309, 50844738, 36963360, 23552601, 27537769, 35221675, 21068623, 2134021], magnitude: 1, normalized: true }, y: Field { n: [42431789, 21934996, 19110004, 16931766, 5594721, 60604108, 39143483, 60081909, 58973295, 2031227], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(0.0)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [60348208, 37111340, 36770446, 46275659, 61457024, 2165878, 60216653, 55793687, 64512720, 2182027], magnitude: 1, normalized: true }, y: Field { n: [23093265, 50267231, 54961297, 62209946, 42641731, 29149859, 6977779, 48468199, 25383024, 4011022], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(323830874445881631945.698587919200669541)) }], accounts: [Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address("f1sdis7xvxkwzr4fd5e4kfnwzaa5efwfsm2e7mvnq")), SignerAddr(Address("f1r6ikc73mxrvi3jakrk2dc5s26y6avuflxvy4opi")), SignerAddr(Address("f17abot4z6mpb24yjzls77qcicf6y4wp6dffzyhwi")), SignerAddr(Address("f1qwo2w25t6ctxqi47kxvewinqd54j2t46umdbdeq")), SignerAddr(Address("f1wkjo6k7xjordlwd5m4sjfx6nkj4jrsaqtg76pqq"))], threshold: 1, vesting_duration: 18446744073709551615, vesting_start: 17804313737527799485 }), balance: TokenAmount(242024390185079582806.941709291648342497) }, Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address("f1pglu43rqfawza3vxm3splsk32boyy3v7733f7xa")), SignerAddr(Address("f17ywqn6mt3edxa3n3vur3wszqmfcab5xee4p2hwq")), SignerAddr(Address("f12hzi7v73a3fe6nn2n7xvd53heemlfm67edbcbpq")), SignerAddr(Address("f1iki2sgvgyb7xyaycaq4rdqyhczwopwz72v5jsyi"))], threshold: 1, vesting_duration: 5311237559795545285, vesting_start: 6367758269187996241 }), balance: TokenAmount(0.0) }, Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address("f165c6pag6ogdeu4htcnrgqupt6lay32jgls74u7i")), SignerAddr(Address("f1t5bog5urgkeijyddnehxkruwfeulk766gsaba5q")), SignerAddr(Address("f1mlmo5mzf6o2zq4tfau3bxwabswz6xebkbdfj4za")), SignerAddr(Address("f1kf7kphgif54aq2wdachgc66wiinkmd3bvsejgmq")), SignerAddr(Address("f17eavhgwpkhpt4p3axhawqksizgbbgaq5dyndt3a"))], threshold: 4, vesting_duration: 12197415362504881376, vesting_start: 15860344553690277279 }), balance: TokenAmount(284496069177540638259.174305476874786561) }, Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address("f1372lvgxv4xgxoodqa3s32kgxcmusbybm6pfj3qi")), SignerAddr(Address("f1fqg34fumsbhjoo6ddmn73zijoyhyo46qysmd2ny")), SignerAddr(Address("f1itqsxxqawfaet6g4rpuemwzhrbrxjn5xco677cy")), SignerAddr(Address("f1s6h2phwwjfzgwzdzk47w54bcfx6brvc3k55stna")), SignerAddr(Address("f17ae5enhwa5wchwjjr434gwtaxie35n6yo7ks6ei"))], threshold: 1, vesting_duration: 11489338513625454354, vesting_start: 9151042756756153382 }), balance: TokenAmount(0.0) }, Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address("f1rrvk2l26ng5werp3zwciboklumn4xbtl2pj5sda")), SignerAddr(Address("f1hiag3nvxd364nvtqlu3vqiiden2bi365wblhdny")), SignerAddr(Address("f1thtj2minopstx2hfddpczyi6cigrvtguhupwu7i"))], threshold: 2, vesting_duration: 6607730496749865171, vesting_start: 4714802589495518313 }), balance: TokenAmount(77647457177848077556.040570171063058191) }, Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address("f1db7j3u2ldrtuhsrgpqnqdczyhunrpqwm3n2uugy")), SignerAddr(Address("f13danfn66spoggnpmwxzsuqduoubw6djgmovjuta"))], threshold: 2, vesting_duration: 18446744073709551615, vesting_start: 6419876409627678053 }), balance: TokenAmount(324796577614060349745.191933553164029409) }, Actor { meta: Account(Account { owner: SignerAddr(Address("f410fyv76tnhyqn5yswzapfyt6n5hyusowifw64maeai")) }), balance: TokenAmount(157674657032175670043.676376941106896097) }], eam_permission_mode: Unrestricted, ipc: Some(IpcParams { gateway: GatewayParams { subnet_id: SubnetID { root: 16634191194911060781, children: [] }, bottom_up_check_period: 470830588806620975, majority_percentage: 67, active_validators_limit: 73 } }) }